composer/airflow_1_samples/bq_copy_across_locations.py

```diff
@@ -19,7 +19,7 @@
 Lastly, the DAG defines a specific application logger to generate logs.
 
 This DAG relies on three Airflow variables
-(https://airflow.apache.org/concepts.html#variables):
+(https://airflow.apache.org/docs/apache-airflow/stable/concepts/variables.html):
```

Comment:
nit: some links are surrounded by parenthesis `(https://...)` and some are not :C

Reply:
lol I have no idea why that one has parens. I just did a find and replace but I'll get rid of the ones in this sample

Reply:
Thank you!! I can sleep in peace 😴🌙